app/views/organizations/_sidebar_additional.html.erb

```diff
@@ -7,12 +7,14 @@
       </div>
     <% end %>
 
+    <%# it's highly unlikely a single org will have so many users to slow down
+        this fetch, thus we can safely use the .present?/.each pattern to load them %>
     <% if @organization.users.present? %>
       <div class="widget">
         <div class="widget-suggested-follows-container">
           <header><h4>meet the team</h4></header>
           <div class="widget-body">
-            <% @organization.users.find_each do |user| %>
```

Comment:
I think this may happen sooner than you think. You get one large org added right now and this will break. I think I would prefer to stick with the safety of `find_each` rather than risking it for a small speed up from using `.present?/each`

Reply:
I was wondering about this line yesterday, but then I decided that I like @rhymes change, since the `find_each`'s default batch size is 1000, and it seems very unlikely that we'd have an org with more users than that. And I guess it wouldn't break, "just" consume a lot of memory.

Reply:
Given we have talked about setting up communities for large organizations, our first Github, Microsoft etc will have well over 1k users. As for breaking modes, this will either timeout in which case the page will fail to load OR if we do OOM there is a chance it takes the whole box down with it as I have not seen great memory slice control from Heroku

Reply:
Ok let's revert it, there's a larger issue at stake here which is the fact that we shouldn't do these kind of queries in the views (actually we shouldn't do queries in the views as much as possible :D). I'm going to switch back to `.find_each` here

Reply:
I saw that you re-requested my review but still seeing the each change here. Did you forget to push it up?

Reply:
@mstruve oh shoot! yes :( done

Reply:
Been to that rodeo before! 😝

Reply:
@mstruve Thanks for the context, I wasn't aware of any plans to onboard organizations of this size.

One question regarding the failure modes: in order to render the page, we'll still need to finish all iterations of `find_each`, so if that takes too long we will still run into timeouts, right? This suggests that we should probably keep a mental note of this particular line and move the SQL queries out of the view as @rhymes suggested.

Reply:
For sure but I think thats a project for another day